extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -198,13 +198,13 @@ describe('ProjectsController', function (): void {
 				const setupResult = await setupDeleteExcludeTest(proj);
 				const scriptEntry = setupResult[0], projTreeRoot = setupResult[1], preDeployEntry = setupResult[2], postDeployEntry = setupResult[3], noneEntry = setupResult[4];
 
-				const projController = new ProjectsController(new SqlDatabaseProjectTreeViewProvider());
+				const projController = new ProjectsController();
 
-				await projController.delete(projTreeRoot.children.find(x => x.friendlyName === 'UpperFolder')!.children[0] /* LowerFolder */);
-				await projController.delete(projTreeRoot.children.find(x => x.friendlyName === 'anotherScript.sql')!);
-				await projController.delete(projTreeRoot.children.find(x => x.friendlyName === 'Script.PreDeployment1.sql')!);
-				await projController.delete(projTreeRoot.children.find(x => x.friendlyName === 'Script.PreDeployment2.sql')!);
-				await projController.delete(projTreeRoot.children.find(x => x.friendlyName === 'Script.PostDeployment1.sql')!);
+				await projController.delete(createWorkspaceTreeItem(projTreeRoot.children.find(x => x.friendlyName === 'UpperFolder')!.children[0]) /* LowerFolder */);
```

Comment:
last night I was playing with this and I did exactly the same thing, even the method name is the same. LOL